sqllineage/config.py

```diff
@@ -1,28 +1,77 @@
 import os
+from dataclasses import dataclass
+from typing import Any, Union
+
+
+@dataclass
+class SQLLineageConfigValue:
+    class_type: Any
+    value: Union[str, bool, None]
+
+
+@dataclass
+class SQLLineageConfigDef:
+    DIRECTORY: SQLLineageConfigValue
+    DEFAULT_SCHEMA: SQLLineageConfigValue
+    TSQL_NO_SEMICOLON: SQLLineageConfigValue
 
 
 class _SQLLineageConfigLoader:
     """
-    Load all configurable items from environment variable, otherwise fallback to default
+    Load all configurable items from config variable, otherwise fallback to default
     """
 
-    # inspired by https://github.com/joke2k/django-environ
-    config = {
-        # for frontend directory drawer
-        "DIRECTORY": (str, os.path.join(os.path.dirname(__file__), "data")),
-        # set default schema/database
-        "DEFAULT_SCHEMA": (str, ""),
-        # to enable tsql no semicolon splitter mode
-        "TSQL_NO_SEMICOLON": (bool, False),
-    }
-
-    def __getattr__(self, item):
-        if item in self.config:
-            type_, default = self.config[item]
-            # require SQLLINEAGE_ prefix from environment variable
-            return type_(os.environ.get("SQLLINEAGE_" + item, default))
+    def __init__(self) -> None:
+        self._config = SQLLineageConfigDef(
+            DIRECTORY=SQLLineageConfigValue(
+                str, os.path.join(os.path.dirname(__file__), "data")
+            ),
+            DEFAULT_SCHEMA=SQLLineageConfigValue(str, None),
+            TSQL_NO_SEMICOLON=SQLLineageConfigValue(bool, False),
+        )
```

Comment:
DO NOT make breaking change. Still try read from environment variable first, if not found then fallback to default value.